src/js/receipt.js

```diff
@@ -6,12 +6,15 @@ import Amount from './amount';
 import utils from './common/utils';
 import EncodeType from './encode-type';
 import aes from './aes';
+import JsBarcode from 'jsbarcode';
+import QRCode from 'qrcode';
 
 const defaultInvoiceAttr = {
-    xmlns: 'urn:GEINV:eInvoiceMessage:C0401:3.1',
-    'xmlns:xsi': 'http://www.w3.org/2001/XMLSchema-instance',
-    'xsi:schemaLocation': 'urn:GEINV:eInvoiceMessage:C0401:3.1 C0401.xsd'
-};
+        xmlns: 'urn:GEINV:eInvoiceMessage:C0401:3.1',
+        'xmlns:xsi': 'http://www.w3.org/2001/XMLSchema-instance',
+        'xsi:schemaLocation': 'urn:GEINV:eInvoiceMessage:C0401:3.1 C0401.xsd'
+    },
+    renderQRcode = Symbol('renderQRcode');
```

Comment:
Could you try to implement a private method by declaring it out of class? If this way can work, I think it is better than Symbol because it is too new to not supported on all browsers. Thanks.

Reply:
Use closure implementation private method.